sourcetool/pkg/policy/policy.go

```diff
@@ -130,6 +131,23 @@ func getLocalPolicy(path string) (*RepoPolicy, string, error) {
 	return &p, path, nil
 }
 
+func (pe PolicyEvaluator) GetRepositoryPolicy(ctx context.Context, slug string) (*RepoPolicy, error) {
+	parts, err := vcslocator.Locator(slug).Parse()
+	if err != nil {
+		return nil, fmt.Errorf("parsing vcs locator: %w", err)
+	}
+	pathbits := strings.Split(strings.TrimSuffix(strings.TrimPrefix(parts.RepoPath, "/"), "/"), "/")
+	if len(pathbits) != 2 {
+		return nil, errors.New("unable to parse repository path to owner/repo")
+	}
+
+	policy, _, err := pe.getPolicy(ctx, ghcontrol.NewGhConnection(pathbits[0], pathbits[1], parts.RefString))
```

Comment:
Optional/Suggestion: make pe.getPolicy public?  It looks like when you call GetRepositoryPolicy you don't have the slug anyways and need to assemble the slug just so you can call this with org/repo.  If GetPolicy were public then you wouldn't need to do that extra work?

Eventually converting a slug to org/repo would be valuable logic, but probably not just here.  I bet we'd want that logic elsewhere.

Reply:
That parse was intended to be just temporary, but I'm reconsidering this: I was thinking we would need a set of public apis that take the slug as I expect it to come from the command line, but thinking about it, I want to move the CLI to prefer input as slugs, but we will need to parse the slug early to assign it to the flag values.

Reply:
OK, I dropped the function in favor of exposing GetPolicy, I've updated the tests too